content/docs/command-reference/fetch.md

```diff
@@ -179,7 +179,7 @@ Note that the `.dvc/cache` directory was created and populated.
 
 Used without arguments (as above), `dvc fetch` downloads all files and
 directories needed by all `dvc.yaml` and `.dvc` files in the current branch. For
-example, the hash values `3863d0e...` and `42c7025...` correspond to the
+example, the hash values `20b786b...` and `c8d307a...` correspond to the
```

Comment:
Where is this `c8d307a` coming from? Why doesn't it match the cache tree above? It should be `32b715e`

Reply:
according to the cache tree shown above.

Reply:
`tree` in line 163 above is not displayed full. `c8d307a..` is at bottom of tree (complete `tree` is displayed on line 251).
Hash value of `model.pkl` can be confirmed from [lock file](https://github.com/iterative/example-get-started/blob/master/dvc.lock). 
I change put `c8d307a` in the `tree` in line 163 instead of `3863d0e`, but that will we wrong order(tree is printed in different order).

Reply:
> tree in line 163 above is not displayed full

O I get that but the point of displaying those 2 cache entries originally is that they would match these 2 files in this paragraph. So just change the entries displayed in that tree so they match please.